pkg/build/controller/build/build_controller.go

```diff
@@ -414,6 +414,7 @@ func (bc *BuildController) cancelBuild(build *buildapi.Build) (*buildUpdate, err
 	podName := buildapi.GetBuildPodName(build)
 	err := bc.podClient.Pods(build.Namespace).Delete(podName, &metav1.DeleteOptions{})
 	if err != nil && !errors.IsNotFound(err) {
+		//TODO - buildapi.StatusReasonCancelBuildFailed is not currently used ... should it be used here?
```

Comment:
yeah probably.

Reply:
actually in revisiting usage of `transitionToPhase` I'm not so sure ... it would change the behavior as to when the error was returned

i'm going to remove the comment, not rock the boat here